setup.py

```diff
@@ -111,7 +145,7 @@ def run(self):
     include_package_data=True,
     license="MIT",
     classifiers=[
-        "Development Status :: 2 - Pre-Alpha",
+        "Development Status :: 5 - Production/Stable",
```

Comment:
This has been incorrect since version 2.0.0 release. Allow me to include this fix.